graylog2-web-interface/package.json

```diff
@@ -61,8 +61,9 @@
     }
   },
   "dependencies": {
+    "@react-bootstrap/pagination": "^1.0.0",
     "bluebird": "^3.4.0",
-    "bootstrap": "^3.4.0",
+    "bootstrap": "^3.3.0",
```

Comment:
https://react-bootstrap-v3.netlify.com/getting-started/introduction is built targeting v3.3 so I wanted to make sure we were rendering the proper styles it was expecting.

Reply:
Version 3.4.0 fixed some XSS issues. I don't think they affect us, but I guess people will get warnings from automated security scans if we downgrade. Since Bootstrap v3 is in maintenance mode and only security issues are being fixed at the moment, I guess it's safe to keep version 3.4 in `package.json`.